src/elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ * @file Sidebar Additional Tab FTUX tooltip
+ * @author Box
+ */
+
+import * as React from 'react';
+
+import TargetedClickThroughGuideTooltip from '../../../features/targeting/TargetedClickThroughGuideTooltip';
+
+import type { UseTargetingApi } from '../../../features/targeting/types';
+
+import './AdditionalTabFtuxTooltip.scss';
+
+type Props = {
+    children: React.Node,
+    targetingApi: UseTargetingApi,
+    text: string,
+};
+
+const AdditionalTabFtuxTooltip = ({ children, targetingApi, text }: Props) => (
+    <TargetedClickThroughGuideTooltip
+        className="bdl-AdditionalTabFtuxTooltip"
+        body={text}
+        position="middle-right"
```

Comment:
Does this work if the sidebar is collapsed?

Reply:
Yep! It shifts over to the left side:
<img width="340" alt="Screen Shot 2020-07-21 at 12 00 38 PM" src="https://user-images.githubusercontent.com/21277746/88095361-d5e7a280-cb49-11ea-9917-03a3b206793e.png">